node_modules/gh-users/tests/util.js

```diff
@@ -38,6 +38,41 @@ var assertGetMe = module.exports.assertGetMe = function(client, callback) {
 };
 
 /**
+ * Assert that a user can be retrieved
+ *
+ * @param  {RestClient}         client                          The REST client to make the request with
+ * @param  {Number}             id                              The id of the user to retrieve
+ * @param  {Function}           callback                        Standard callback function
+ * @param  {User}               callback.user                   The retrieved user
+ * @throws {AssertionError}                                     Error thrown when an assertion failed
+ */
+var assertGetUser = module.exports.assertGetUser = function(client, id, callback) {
+    client.user.getUser(id, function(err, user) {
+        assert.ok(!err);
+        assert.ok(user);
```

Comment:
Could optionally pass the expectedUser in here and compare all of the properties. Should verify that the basic properties are present regardless